spec/graphql/schema/build_from_definition_spec.rb

```diff
@@ -69,6 +69,8 @@ def build_schema_and_compare_output(definition)
 
 # What a great type
 type Hello {
+  anEnum: Color
```

Comment:
Any idea why the Enum was printed before if it wasnt referenced by anything? 🤔

Reply:
Probably the way build_from_definition works :/

Reply:
Woah, good find! Yeah, I guess because it takes _all_ the types in the `.graphql` and passes them as `orphan_types`: https://github.com/rmosolgo/graphql-ruby/blob/master/lib/graphql/schema/build_from_definition.rb#L100